src/NHSD.BuyingCatalogue.Organisations.Api/Controllers/UsersController.cs

```diff
@@ -63,9 +68,14 @@ public async Task<ActionResult> CreateUserAsync(Guid organisationId, CreateUserR
                 OrganisationFunction = "Buyer",
                 CatalogueAgreementSigned = false
             };
-            
+
             await _usersRepository.CreateUserAsync(newApplicationUser);
 
+            // TODO: discuss exception handling options 
+            // TODO: consider moving sending e-mail out of process
+            // (the current in-process implementation has a significant impact on response time)
```

Comment:
I agree with the comments but it might be more of a style thing but I would rather raise a task on DevOps than leave the TODO comment within the application. I have seen TODO comments just get ignored & left in an application for years (You could say this about some DevOps tasks). At least more roles within the team would see we are introducing tech debt by doing it this way.

I mentioned it to @michaelpace-bjss so he is aware of it. Agreed that for now we will leave it how you've implemented it, but will need to change it soon.

Reply:
I don't think there's any harm in having the TODO comments given that a change is always reviewed by at least one person. While I agree that there should be a corresponding task having the TODO comment means that it should prompt discussions. It's also a reminder to add a task (or check that a task has been added) should I (or someone else) forget to add one.

If the TODO comments are still there the next time someone else works on the code, hopefully it will prompt them to ask: does this still need doing? I also like that they appear in the task list in VS although I'm not sure how many people are familiar with that feature.

I'll leave them in for now but we can discuss at our next code standards meeting.

The corresponding tasks in this case are 5718 and 5720.